### ISSUE TITLE
tagging of recipes

### ISSUE THREAD

**Issue by everycook** [everycook/EveryCook#210]
tag input field must either be larger or grow when adding tags.

**Comment by everycook** [comment]
show tags instead of cuisine in overview

**Comment by everycook** [comment]
it would be more user friendly to show existing tags on top of the entered string. also helps people to use existing tags instead of creating new ones

**Comment by swerder** [comment]
field grows now also on chrome.

**Comment by tbuerki** [comment]
I agree tags should be added to overview but we don't replace the cuisine, it will still be there. Also, I wouldn't change yet whether tags are shown above or below, the UX details should be decided at the end otherwise we change it back and forth.

**Comment by everycook** [comment]
Hearachical Cuisine won't work. That's why nobody is doing it. We can use the data sructure to improve the tagging system. that's all.

**Comment by tbuerki** [comment]
Ok, then let's do both (add tags and keep cuisine) for the moment until we have fully decided on that (removing is always easy)
